third_party/move/move-compiler-v2/tests/live-var/mut_ref.exp

```diff
@@ -162,14 +162,6 @@ fun m::some2($t0: &mut m::R, $t1: &mut m::R) {
   0: return ()
 }
 
-
-Diagnostics:
-error: implicit copy of mutable reference in local `x` which is used later in argument list
-   ┌─ tests/live-var/mut_ref.move:41:9
-   │
-41 │         some2(x, x); // expected error because multiple use
```

Comment:
We seem to be losing this error in multiple test cases.

Reply:
Livevar doesn't longer check this, so its legit.

Reply:
It is now in reference_safety/multiple_use_invalid, and also a bit more extensively tested, specifically involving reference aliasing.

Reply:
Ok.